### DISCUSSION TITLE
Broken link of Contributor Guide in Readme

### DISCUSSION THREAD

**Discussion by nirban07** [wagtail/gsoc#121] [General]
Hi, I was going through the readme and found that theres a broken link when we click on contributor guide in the Getting Started part of the Readme. I didnt see the issues tab so i opened this discussion here.

**Comment by p0lygun** [comment]
in 2023 it was linked to gsoc specific contributor guide, it seemed to have been removed for 2024 :thinking:

**Comment by thibaudcolas** [comment]
Thanks! I’ve fixed the link to the 2023 guide. For 2024 I’ve removed it because we don’t have capacity to do much beyond the baseline defined by Google. If it helps, here’s the official [GSoC contributor guide](https://google.github.io/gsocguides/student/).
